VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -226,7 +226,7 @@ private void restore(Bundle savedInstanceState) {
                 showVideoNotAvailable(VideoNotPlayMessageType.IS_VIDEO_MESSAGE_DISPLAYED);
             }
         } else {
-            player = new Player();
+            if (player == null) player = new Player();
```

Comment:
This seems quite a strange issue, as for some reason `onActivityCreated()` is called multiple times by the `FragmentManager`, sometimes with `null` `savedInstanceState`. But I guess this fix should work.